new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -65,69 +64,89 @@ public void testInstallingOperator() {
     // the kubernetes deployment.  this will complete quickly, and will either be
     // successful or not.
 
-    // get a new unique namespace
-    final String namespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+    // get a new unique opNamespace
+    final String opNamespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", opNamespace));
+
+    final String domainNamespace1 = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", domainNamespace1));
+
+    final String domainNamespace2 = assertDoesNotThrow(TestActions::createUniqueNamespace,
         "Failed to create unique namespace due to ApiException");
-    logger.info(String.format("Got a new namespace called %s", namespace));
+    logger.info(String.format("Got a new namespace called %s", domainNamespace2));
 
-    // Create a service account for the unique namespace
-    final String serviceAccountName = namespace + "-sa";
-    assertDoesNotThrow(
-        () -> TestActions.createServiceAccount(new V1ServiceAccount()
-            .metadata(new V1ObjectMeta().namespace(namespace).name(serviceAccountName))));
+    // Create a service account for the unique opNamespace
+    final String serviceAccountName = opNamespace + "-sa";
+    assertDoesNotThrow(() -> TestActions.createServiceAccount(new V1ServiceAccount()
+        .metadata(new V1ObjectMeta().namespace(opNamespace).name(serviceAccountName))));
     logger.info("Created service account: " + serviceAccountName);
 
     OperatorParams opParams =
         new OperatorParams().releaseName("weblogic-operator")
-            .namespace(namespace)
-            .image("weblogic-kubernetes-operator:test_itsimpleoperator")
-            .domainNamespaces(Arrays.asList("domainns1", "domainns2"))
-            .serviceAccount("opns1-sa");
+            .namespace(opNamespace)
+            .chartDir("../kubernetes/charts/weblogic-operator")
+            .image("phx.ocir.io/weblogick8s/weblogic-kubernetes-operator:develop")
```

Comment:
This is the part I was discussing in the channel... Depending on the sort of cluster, the image may need to be pushed or not.  But, I guess this can wait and need not be part of this pull request.